bigquery_etl/pytest_plugin/sql.py

```diff
@@ -71,27 +71,20 @@ def runtest(self):
         """Run."""
         test_name = self.fspath.basename
         query_name = self.fspath.dirpath().basename
-        dataset_name = self.fspath.dirpath().dirpath().basename
-        project_name = self.fspath.dirpath().dirpath().dirpath().basename
         project_dir = (
             self.fspath.dirpath().dirpath().dirpath().dirname.replace("tests", "")
         )
 
-        init_test = False
         script_test = False
 
         # init tests write to dataset_query_test, instead of their default name
         path = self.fspath.dirname.replace("tests", "")
         if test_name == "test_init":
```

Comment:
There are some references to init.sql in docs/cookbooks/testing.md that can be updated